helm/cas-airflow/.helmignore

```diff
@@ -0,0 +1 @@
+templates/rbac/pod-launcher-role.yaml
```

Comment:
Why is this ignored?

Reply:
this is a remnant of us playing with cas-pipeline roles, it's not needed anymore since we're using `cas-provision-job-runner` instead

Reply:
I'll delete it

Reply:
I think that you deleted the file but forgot to remove this line

Reply:
oops, I meant to remove helmignore entirely, it's not needed anymore